pkg/util/validation/field/path.go

```diff
@@ -67,9 +67,14 @@ func (p *Path) Key(key string) *Path {
 
 // String produces a string representation of the Path.
 func (p *Path) String() string {
+	return p.StringRelative(nil)
+}
+
+// String produces a string representation of the Path.
```

Comment:
OneOf or any other validator may get a list of fields that could be in one type or different sub-types. For example I may want to anyobject.meta.name and anyobject.custom.name and anyobject.name be passed to a validator. The validator can use these methods to find the common parent (anyobject) and refer to those parameters relative to the common parent. e.g., "validation failed for anyobject [common parent], meta.name [relative path], custom.name[relative path] and name[relative path] need to be ..."